java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/deploy/DeployCommand.java

```diff
@@ -1,108 +1,52 @@
 package com.redhat.hacbs.container.deploy;
 
-import static com.redhat.hacbs.classfile.tracker.TrackingData.extractClassifier;
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
 import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
 
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.FileVisitResult;
-import java.nio.file.Files;
+import java.net.URL;
 import java.nio.file.Path;
-import java.nio.file.SimpleFileVisitor;
-import java.nio.file.attribute.BasicFileAttributes;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
 import java.util.Optional;
-import java.util.Set;
-import java.util.function.BiConsumer;
+import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import java.util.stream.Stream;
 
-import jakarta.enterprise.inject.spi.BeanManager;
+import jakarta.inject.Inject;
 
-import org.apache.commons.lang3.StringUtils;
-import org.cyclonedx.Version;
-import org.cyclonedx.generators.BomGeneratorFactory;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 
-import com.redhat.hacbs.classfile.tracker.ClassFileTracker;
-import com.redhat.hacbs.classfile.tracker.TrackingData;
-import com.redhat.hacbs.common.sbom.GAV;
-import com.redhat.hacbs.container.analyser.dependencies.SBomGenerator;
-import com.redhat.hacbs.container.deploy.containerregistry.ContainerRegistryDeployer;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.auth.profile.ProfileCredentialsProvider;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.codeartifact.AWSCodeArtifactClientBuilder;
+import com.amazonaws.services.codeartifact.model.GetAuthorizationTokenRequest;
+import com.amazonaws.util.AwsHostNameUtils;
 import com.redhat.hacbs.container.deploy.git.Git;
-import com.redhat.hacbs.container.results.ResultsUpdater;
-import com.redhat.hacbs.recipes.util.FileUtil;
-import com.redhat.hacbs.resources.model.v1alpha1.dependencybuildstatus.Contaminates;
-import com.redhat.hacbs.resources.util.HashUtil;
+import com.redhat.hacbs.container.deploy.mavenrepository.CodeArtifactRepository;
+import com.redhat.hacbs.container.deploy.mavenrepository.MavenRepositoryDeployer;
 
+import io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext;
 import io.quarkus.logging.Log;
 import picocli.CommandLine;
 
-@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
 @CommandLine.Command(name = "deploy")
 public class DeployCommand implements Runnable {
 
```

Comment:
The git diff isn't showing this very well. I renamed MavenDeployCommand to DeployCommand and the original DeployCommand to VerifyCommand.